### DISCUSSION TITLE
aiocr

### DISCUSSION THREAD

**Discussion by giscus** [jianchang512/pyvideotrans#693] [Announcements]
# aiocr

免费开源使用简单的视频翻译配音软件-pyVideoTrans

https://pyvideotrans.com/aiocr

**Comment by sailing178** [comment]
我试了一下谭乔的视频“我不想活了"！是什么逼得一位少女绝望高呼？（答案请看简介）.mp4”，发现字幕很多识别不到，甚至出现政治敏感提示。感觉是因为字幕存在视频帧比较少，请查阅

**Reply by sailing178** [reply]
字幕的速度也跟不上视频

**Comment by ai725** [comment]
有木有办法可以提升速度？
